desktop/src/main/java/bisq/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -392,6 +407,19 @@ protected void deactivate() {
     // UI handlers
     ///////////////////////////////////////////////////////////////////////////////////////////
 
+    private void applyFilteredListPredicate(String filterString) {
+        filteredList.setPredicate(item -> {
+            if (filterString.isEmpty())
+                return true;
+
+            if (item.getBalance().toString().contains(filterString)) {
```

Comment:
This does not cover formatted string. E.g. if balance is 0.01 BTC and you type in 0.01 it is not found.
Better create a new string field in the itme (formatter.formatCoin(this.balance) which reflects the displayed string

Reply:
[Fixed](https://github.com/bisq-network/bisq/pull/5963/commits/35f01a6ca3f9a457471d7c9e9eb0a0645d315547)